OpenUpSA/wazimap-ng-ui
# Add check to not display qualitative indicators in datamapper

## Description

HTML/qualitative data was being displayed by indicators on the data mapper, this should not be

## Related Issue
https://trello.com/c/9mTq0Scz/888-qualitative-data-shows-up-in-data-mapper

## How to test it locally
On YE staging profile navigate to the data mapper menu and check that the qualitative indicator is there, see screenshot
For this fix see the same place and confirm the qualitative indicator is not shown

## Screenshots
![qualitativedm](https://user-images.githubusercontent.com/9511724/131519833-0110b3b4-4245-4611-941b-b4f3793c1055.png)

## Changelog

### Added

### Updated
Fixed qualitative indicators appearing in the data mapper
### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
